src/Symfony/Bundle/SecurityBundle/Tests/Templating/Helper/SecurityHelperTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\SecurityBundle\Tests\Templating\Helper;
+
+use Symfony\Bridge\Twig\Extension\SecurityExtension;
+use Symfony\Component\Security\Acl\Voter\FieldVote;
+use Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface;
+
+/**
+ * @author Iltar van der Berg <ivanderberg@hostnet.nl>
+ */
+class SecurityHelperTest extends \PHPUnit_Framework_TestCase
```

Comment:
adding new tests in a PR removing deprecated APIs is wrong. Such PR should only be removing code. Missing tests should be added in the older branch where they are worth it.

Btw, the name and namespace of the test is wrong. You are testing ``Symfony\Bridge\Twig\Extension\SecurityExtension``, so this test should be in the bridge and be named ``SecurityExtensionTest``